components/identifier-generator/front/src/feature/tabs/structure/preview/AutoNumberPreview.tsx

```diff
@@ -9,12 +9,9 @@ type AutoNumberPreviewProps = {
 const AutoNumberPreview: React.FC<AutoNumberPreviewProps> = ({property}) => {
   const {digitsMin, numberMin} = property;
 
-  const getFormattedNumber = useMemo(
-    () => (i: number) => String(Math.max(i, numberMin)).padStart(digitsMin, '0'),
-    [digitsMin, numberMin]
-  );
+  const getFormattedNumber = useMemo(() => (i: number) => String(i).padStart(digitsMin, '0'), [digitsMin]);
```

Comment:
Since all used params are available, it doesn't have to be a function.

```suggestion
  const formattedNumber = useMemo(() => String(numberMin).padStart(digitsMin, '0'), [digitsMin, numberMin]);
```

And below it would be
```
  return <Preview.Highlight>{formattedNumber}</Preview.Highlight>;
```